lib/util.js

```diff
@@ -100,6 +100,12 @@ const util = {
       // With this copying, we don't need to modify chrome/BUILD.gn for this.
       fs.copySync(path.join(braveAppDir, 'theme', 'brave', 'mac', config.channel, 'app.icns'),
                   path.join(chromeAppDir, 'theme', 'brave', 'mac', 'app.icns'))
+
+      // Copy branding file
+      let branding_file_name = 'BRANDING'
```

Comment:
doesn't this already happen somewhere else? Or did that not get transferred over from blb?

Reply:
So far, we only used one kind of BRANDING file. (copied from brave/app/theme/brave/BRANDING to chrome/app/theme/brave/BRANDING.)
To support multiple install on MacOS, different id(MAC_BUNDLE_ID) is needed and it is defined BRANDING file.
For that, I added different BRANDING files(ex, BRANDING.dev in brave folder) and it is copied as BRANDING file.